docs/content/direct/common-setup-intro.md

```diff
@@ -0,0 +1,16 @@
+# KubeStellar Hub Setup
+
+This document is is excerpted from [our example scenarios](examples.md) document to show two ways of deploying the kubestellar hub (core) components to create the "common setup" used by those examples.
+
+
+## Common Setup
+
+The following steps set up the hub and establish its initial state
+
+There are two main ways to proceed:
+
+2. [Step-by-step manual setup](common-setup-step-by-step.md)
+1. [Automatically Using KubeStellar Helm chart](core-chart.md)<br/>
```

Comment:
Before this PR, there are two files that serve different purposes:
- one that is the draft of the new contents for examples.md, showing the "getting started" or "quickstart" version of using the core chart; and
- core-chart.md, the user guide contents about the core chart.

I think that it makes sense to not conflate the two. The plan was to move the first file inline into examples.md once @francostellari was sufficiently confident about it; the "step-by-step" alternative is just an interim thing until then --- the plan was that it gets deleted by the time we make the next release.

Reply:
The introduction of the core Helm chart emphasizes this, but it was already true: setting up KubeStellar does not require creating a hosting cluster --- a pre-existing cluster could be used. The quickstart begins by creating a hosting cluster using `kind` (indirectly through `kflex init --create-kind`), but that is not the only way. Really there are three steps to discern here:

1. Acquire the use of a cluster (create one or get credentials to use an existing one).
2. Initialize it for use as a KubeFlex hosting cluster.
3. Install the KubeStellar core components in this cluster.